src/main/java/com/EstimatedDeliveryDate/Repository/CarrierDataService.java

```diff
@@ -0,0 +1,12 @@
+package com.EstimatedDeliveryDate.Repository;
+
+import com.EstimatedDeliveryDate.Entity.CriticalPullTime;
+import com.EstimatedDeliveryDate.Entity.Holidays;
+import com.EstimatedDeliveryDate.Entity.TransitTime;
+
+public interface CarrierDataService {
```

Comment:
Does renaming the class to EstimatedDeliveryDateRepository and implementation to EstimatedDeliveryDateRepositoryImpl sound better?

Reply:
Ok I will change that